test/parallel/test-console-formatTime.js

```diff
@@ -4,12 +4,11 @@ require('../common');
 const { formatTime } = require('internal/console/constructor');
 const assert = require('assert');
 
-const test1 = formatTime(100);
-const test2 = formatTime(1500);
-const test3 = formatTime(60300);
-const test4 = formatTime(4000000);
-
-assert.strictEqual(test1, '100.000ms');
-assert.strictEqual(test2, '1.500s');
-assert.strictEqual(test3, '1.005min');
-assert.strictEqual(test4, '1.111h');
+assert.strictEqual(formatTime(100.0096), '100.01ms');
+assert.strictEqual(formatTime(100.0115), '100.011ms');
+assert.strictEqual(formatTime(1500.04), '1.500s');
+assert.strictEqual(formatTime(1000.056), '1.000s');
```

Comment:
I would keep these consistent with the higher numbers, to be honest...

e.g.
```suggestion
assert.strictEqual(formatTime(1000.056), '1.000 (ss.mmm)');
```

Reply:
Seconds and milliseconds are the most common ones. People will rarely reach the higher values and it seems more straight forward to me to keep `s` and `ms` as common cases.

Reply:
Yeah I understand the reasoning, I just disagree :) I'd rather opt for consistency.

Reply:
I'm going to remove the `author ready` label out of caution while this gets sorted, but feel free to re-add it if that's overly-cautious of me.

Reply:
@jasnell is your comment blocking? I would rather close this PR than changing the common cases (even though I believe that this PR does improve the overall output).

Reply:
Not blocking but I definitely don't consider the inconsistency to be ideal.